engines/tinsel/dialogs.cpp

```diff
@@ -5214,6 +5221,24 @@ void Dialogs::idec_inv2(SCNHANDLE text, int MaxContents,
 	         100, 100, true);
 }
 
+/**
+ * Called from Glitter functions: dec_invMain
+ * - Declare inventories 1,3 and 4 and hail the loadingscreen(?) scene.
+ */
+void Dialogs::idec_invMain(SCNHANDLE text, int MaxContents) {
+	idec_inv(INV_1, text, MaxContents,3, 2, 3, 2, 3, 2, 39,
+			72, false);
+	idec_inv(INV_MENU, 0, 3, 2, 2, 2, 1, 3, 1, 100, 100,
+			false);
+	idec_inv(INV_4, text, MaxContents,3, 2, 3, 2, 3, 2, 39,
```

Comment:
This goes out of bounds currently, as Noir has a larger _invD (6) than the earlier games.